src/client/helpers/utils.js

```diff
@@ -34,4 +34,24 @@ function injectDefaultAliasName(instance) {
 	return instance;
 }
 
+function formatDate(date, includeTime) {
+	'use strict';
+	if (!date || !(Object.prototype.toString.call(date) === '[object Date]') ||
```

Comment:
I'm not clear on what the second condition here is trying to accomplish. Is this going for `date instanceof Date`?